packages/client/eslint.config.js

```diff
@@ -0,0 +1,83 @@
+import path from "node:path"
+import { fileURLToPath } from "node:url"
+
+import { fixupConfigRules, fixupPluginRules } from "@eslint/compat"
+import { FlatCompat } from "@eslint/eslintrc"
+import js from "@eslint/js"
+import typescriptEslint from "@typescript-eslint/eslint-plugin"
+import tsParser from "@typescript-eslint/parser"
+import react from "eslint-plugin-react"
+import simpleImportSort from "eslint-plugin-simple-import-sort"
+import globals from "globals"
+
+const __filename = fileURLToPath(import.meta.url)
+const __dirname = path.dirname(__filename)
+const compat = new FlatCompat({
+  baseDirectory: __dirname,
+  recommendedConfig: js.configs.recommended,
+  allConfig: js.configs.all,
+})
+
+const fixedUpRuleExtensions = fixupConfigRules(
+  compat.extends(
+    "eslint:recommended",
+    "plugin:react/recommended",
+    "plugin:react/jsx-runtime",
+    "plugin:@typescript-eslint/recommended",
+    "prettier",
```

Comment:
does this make eslint play nice with prettier? Is this to be able to use prettier as the formatter in vscode?

Reply:
now the extra config negating eslint rules no longer necessary, it seems, as stylistic rules have been deprecated in these recent versions of the linting packages .. happy days 🙂